.travis.yml

```diff
@@ -26,11 +22,10 @@ after_success:
   fi
   if [[ $TRAVIS_BRANCH == 'next' && $TRAVIS_PULL_REQUEST == 'false' ]]; then
     # TODO: check changed path to decide if npm publish necessary
-    npm run link;
-    lerna bootstrap;
     echo //registry.npmjs.org/:_authToken=${NPM_TOKEN} > ~/.npmrc;
-    lerna publish -c --npm-tag next --yes --exact;
-    npm run deploy && echo 'Website updated.';
+    # TODO: implement publish script @gaspard
```

Comment:
don't hurry with publish script. there are a lot of details we must consider. let's finish this step first